<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="framefondo">
      <property name="styleSheet">
       <string notr="true">QWidget#framefondo{
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0.039548 rgba(0, 67, 109, 255), stop:1 rgba(203, 203, 203, 255));}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_superior">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>0</y>
         <width>911</width>
         <height>40</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 85, 127);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>10</y>
          <width>331</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Arial&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>PROPIETARIOS</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="btnPropiedades">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>80</y>
         <width>241</width>
         <height>61</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton{
font: 87 9pt &quot;Arial&quot;;
background-color: rgb(0, 68, 100);
color:white;

border-radius: 10px;
}


QPushButton:hover {
background-color:rgb(0, 101, 148);
}</string>
       </property>
       <property name="text">
        <string>PROPIEDADES </string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>./Imagenes/casa.png</normaloff>./Imagenes/casa.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="btnINQUILINOS">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>150</y>
         <width>241</width>
         <height>61</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 87 9pt &quot;Arial&quot;;
background-color: rgb(0, 68, 100);
color:white;

border-radius: 10px;
}


QPushButton:hover {
background-color:rgb(0, 101, 148);
}</string>
       </property>
       <property name="text">
        <string>INQUILINOS </string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../PROYECTO 1/Interfaz Grafica/Imagenes/inquilino.png</normaloff>../../PROYECTO 1/Interfaz Grafica/Imagenes/inquilino.png</iconset>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>180</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>160</y>
         <width>201</width>
         <height>191</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../PROYECTO 1/Interfaz Grafica/Imagenes/propietarios.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="btnMANTENIMIENTO">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>220</y>
         <width>241</width>
         <height>61</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 87 9pt &quot;Arial&quot;;
background-color: rgb(0, 68, 100);
color:white;

border-radius: 10px;
}


QPushButton:hover {
background-color:rgb(0, 101, 148);
}</string>
       </property>
       <property name="text">
        <string>MANTENIMIENTO </string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../PROYECTO 1/Interfaz Grafica/Imagenes/mantenimiento.png</normaloff>../../PROYECTO 1/Interfaz Grafica/Imagenes/mantenimiento.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="btnCOMUNICACION">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>290</y>
         <width>241</width>
         <height>61</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 87 9pt &quot;Arial&quot;;
background-color: rgb(0, 68, 100);
color:white;

border-radius: 10px;
}


QPushButton:hover {
background-color:rgb(0, 101, 148);
}</string>
       </property>
       <property name="text">
        <string>COMUNICACIÃ“N</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../PROYECTO 1/Interfaz Grafica/Imagenes/mjs.png</normaloff>../../PROYECTO 1/Interfaz Grafica/Imagenes/mjs.png</iconset>
       </property>
      </widget>
      <widget class="QPushButton" name="btnREPORTE">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>360</y>
         <width>241</width>
         <height>61</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
font: 87 9pt &quot;Arial&quot;;
background-color: rgb(0, 68, 100);
color:white;

border-radius: 10px;
}


QPushButton:hover {
background-color:rgb(0, 101, 148);
}</string>
       </property>
       <property name="text">
        <string>REPORTE </string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Imagenes/reporte.png</normaloff>Imagenes/reporte.png</iconset>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
